src/Model/ModelManager.php

```diff
@@ -148,9 +158,19 @@ public function hasMetadata($class)
 
     /**
      * @psalm-suppress InvalidArgument
+     *
+     * @deprecated since sonata-project/doctrine-mongodb-admin-bundle 3.x and will be removed in version 4.0
```

Comment:
```suggestion
     * @deprecated since sonata-project/doctrine-mongodb-admin-bundle 3.x and will be removed in version 4.0
```
```suggestion
     * @deprecated since sonata-project/doctrine-mongodb-admin-bundle 3.x and will be removed in version 4.0.
```